### ISSUE TITLE
Some Characters are not displayed as it was when pasted

### ISSUE THREAD

**Issue by aditya-mitra** [max-programming/easypastes#6]
**Describe the bug**

Some characters like the `>=` and the `!=` are displayed in some other format.
When I share these pastes to other people, they find it difficult to decipher these characters.

![image](https://user-images.githubusercontent.com/55396651/129069401-e692bc37-9fcb-42d9-86bb-cb3767bb396d.png)

![image](https://user-images.githubusercontent.com/55396651/129069423-0c7a95df-5400-4bef-8c96-28066579ab83.png)

**To Reproduce**

1. paste the code below
2. submit in easypastes

```

 while(start < end){
            if(str[start++] != str[end--]){
                return false;
            }
        }
        return true;
    }

    void backtrack(vector<string> &curr, const int start){
        if(start >= str.length()){
            ans.push_back(curr);
        }
```

**Expected behavior**

The `!=` character should be displayed just like the **`!=`** character.

**Screenshots**

![image](https://user-images.githubusercontent.com/55396651/129069029-2300dc72-8628-4197-beb8-1b8f44bb1c95.png)

https://www.easypastes.tk/pastes/2z4pV5

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**
 - Browser [e.g. stock browser, safari] any

**Additional context**

**Comment by janaSunrise** [comment]
This is intended behaviour. We use Fira code font for the ligatures. If you're not sure about what ligatures are, Check [this out](https://github.com/tonsky/FiraCode#whats-in-the-box).

It will be copied as normal text, for example if you copy not equal to symbol, It is copied as `!=` regardless of the unicode ligature display.

Hope this helps you! 😁

**Comment by aditya-mitra** [comment]
@janaSunrise Can you put some setting for a particular account so that this setting can be turned off for that account only?
**The problem is** some people misunderstand the character as some other literal in the language.
Thanks.

**Comment by janaSunrise** [comment]
We'll look into this issue, once we get the time! Since this is more of a feature addition.

**Comment by max-programming** [comment]
Yes we'll add an option for disabling font ligatures. Thanks for creating the issue @aditya-mitra

**Comment by janaSunrise** [comment]
@max-programming We could add a toggle button to enable or disable ligatures, Connected to DB. Just like how we have for other options. More about that later, Adding this PR to the TODO list, will be worked on when we are done with the list, as we are going through each feature one by one.

**Comment by max-programming** [comment]
> @max-programming We could add a toggle button to enable or disable ligatures, Connected to DB. Just like how we have for other options. More about that later, Adding this PR to the TODO list, will be worked on when we are done with the list, as we are going through each feature one by one.
> 
> Making it a toggle button based config on a paste basis, could enable user to customize each paste how they want. It would make it easier in my opinion too. We will work on this once we tackle the other features one by one. Hope you understand! 😁

Yes. Or we could create a modal for **additional settings** where we could add this and many other settings
